spec/lib/miq_automation_engine/miq_ae_event_spec.rb

```diff
@@ -99,6 +99,21 @@ module MiqAeEventSpec
 
           MiqAeEvent.raise_evm_event("host_provisioned", host, :host => host)
         end
+
+        it "has tenant from root tenant" do
+          host = FactoryGirl.create(:host, :ext_management_system => ems)
+          ems.destroy
+          host.reload
```

Comment:
@lfu Is there any reason to create the host with an EMS and then delete the EMS?  You should be able to just create the host and not pass any EMS and avoid the `destroy` and `reload`.